src/Symfony/Component/Messenger/Bridge/AmazonSqs/composer.json

```diff
@@ -19,7 +19,7 @@
         "php": ">=7.2.5",
         "async-aws/core": "^1.5",
         "async-aws/sqs": "^1.0|^2.0",
-        "symfony/messenger": "^4.3|^5.0|^6.0",
+        "symfony/messenger": "^5.4|^6.0",
```

Comment:
I think you must use the next 5.4 patch version, and also increase  6.0 to the next 6.4 patch version

Reply:
Indeed, fixed

Reply:
With this feedback https://github.com/symfony/symfony/pull/54920#discussion_r1601958526, I don't even need this change ^^'